README.md

```diff
@@ -474,6 +474,38 @@ services:
     restart: unless-stopped
 ```
 
+## Debugging containers
+
+Sometimes it is useful to debug the code which is running inside the container.  Two scripts (*'debug'* and *'debug_brk'* in the package.json file) are available to start NodeJs in debug mode, which means that NodeJs will start listening (to port 9229) for a debug client.
+
+1. In most cases the *'debug'* script will be sufficient, to debug a Node-RED application that is already running.  The NodeJs server can be started in debug mode using following command:
```

Comment:
Does it really debug an already running application ? I think this this just starts and runs the application with the debug port enabled. (IE It doesn't attach to an already running instance ? - or does it ?)

Reply:
That is true.  But what I mean is:
1. NodeJs starts listening to the debug port
1. NodeJs starts Node-RED
1. Node-RED will start the flow
1. Then you connect with your debugger to the debug port, and you can start debugging

About point "1": immediately when the log displays that NodeJs has started listening, you will get the "Inspect" link to open a debugger window:

![image](https://user-images.githubusercontent.com/14224149/74104922-cbb48780-4b59-11ea-8f74-fe988cc8a736.png)

But then you have to add your breakpoints, for which you don't have time (since 1 to 3 happens very fast).  So this is only useful when you want to start debugging once Node-RED is up and running...

Perhaps I should say "_to debug a Node-RED application once it is fully started_"?

Reply:
maybe just ... "to debug a Node-RED application using a remote debugger tool like Visual Code or Chrome dev tools." ?

Reply:
Yes but the ***"once it is fully started"*** is important, because the seconds script also supports all those tools...  I have rewritten the readme page a bit, and also added the link to the wiki page.